doc/whats_new.rst

```diff
@@ -44,7 +46,7 @@ BUG
 API
 ~~~
 
-    - None
+    - `mne.io.read_raw_brainvision` now has `event_id` argument to assign non-standard trigger events to a trigger value by `Teon Brooks`_
```

Comment:
I think this needs to be `:func:`mne.io.read_raw_brainvision` to get it to link properly, and you'll want double-backticks on the `event_id` (because Sphinx there won't know where to link it, so better to render it as code instead)